yarn.lock

```diff
@@ -1684,6 +1684,16 @@
     "@types/qs" "*"
     "@types/serve-static" "*"
 
+"@types/express@4.17.8":
+  version "4.17.8"
+  resolved "https://registry.yarnpkg.com/@types/express/-/express-4.17.8.tgz#3df4293293317e61c60137d273a2e96cd8d5f27a"
+  integrity sha512-wLhcKh3PMlyA2cNAB9sjM1BntnhPMiM0JOBwPBqttjHev2428MLEB4AYVN+d8s2iyCVZac+o41Pflm/ZH5vLXQ==
+  dependencies:
+    "@types/body-parser" "*"
+    "@types/express-serve-static-core" "*"
+    "@types/qs" "*"
+    "@types/serve-static" "*"
+
```

Comment:
Lockfile changes indicate to me that something is off about the versions being used in the `package.json` for the API. Can you make sure they're aligned with the versions in other packages?

Reply:
not sure what i'm supposed to do here.  packages look aligned to me, between the Service and the API.

Reply:
Make sure they're aligned with the other API/Service packages too